File: Api/src/Api/Models/Utilisateur.cs
using System;
using System.Collections.Generic;

namespace Api.Models;

public partial class Utilisateur
{
    public int Id { get; set; }

    public int IdGroupe { get; set; }

    public Guid IdPublic { get; set; }

    public string Nom { get; set; } = null!;

    public string Prenom { get; set; } = null!;

    public string Mail { get; set; } = null!;

    public string Mdp { get; set; } = null!;

    public string? Telephone { get; set; }

    public bool EstAdmin { get; set; }

    public virtual Groupe IdGroupeNavigation { get; set; } = null!;

    public virtual ICollection<PlanningProduitUtilisateur> PlanningProduitUtilisateurs { get; set; } = new List<PlanningProduitUtilisateur>();
}

File: front/src/models/Utilisateur.ts
export type UtilisateurConnecter =
{
    nom: string,
    prenom: string,
    jwt: string
}

export type Utilisateur =
{
    idPublic: string,
    nom: string,
    prenom: string,
    mail: string,
    telephone?: string,
    estAdmin: boolean
}

export type UtilisateurLeger =
{
    idPublic: string,
    nomComplet: string
}